optuna/integration/chainermn.py

```diff
@@ -103,12 +97,11 @@ def __init__(
 
     def optimize(
         self,
-        func,  # type: Callable[[ChainerMNTrial, CommunicatorBase], float]
-        n_trials=None,  # type: Optional[int]
-        timeout=None,  # type: Optional[float]
-        catch=(),  # type: Tuple[Type[Exception], ...]
-    ):
-        # type: (...) -> None
+        func: "Callable[[ChainerMNTrial, CommunicatorBase], float]",
```

Comment:
We can also access `Callable` and `float` here.
```suggestion
        func: Callable[["ChainerMNTrial", "CommunicatorBase"], float],
```